collector/logical_router_port_collector.go

```diff
@@ -0,0 +1,174 @@
+package collector
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	nsxt "github.com/vmware/go-vmware-nsxt"
+	"github.com/vmware/go-vmware-nsxt/manager"
+)
+
+func init() {
+	registerCollector("logical_router_port", newLogicalRouterPortCollector)
+}
+
+type logicalRouterPortCollector struct {
+	client *nsxt.APIClient
+	logger log.Logger
+
+	rxTotalPacketDesc   *prometheus.Desc
+	rxDroppedPacketDesc *prometheus.Desc
+	rxTotalByteDesc     *prometheus.Desc
+	txTotalPacketDesc   *prometheus.Desc
+	txDroppedPacketDesc *prometheus.Desc
+	txTotalByteDesc     *prometheus.Desc
+}
+
+func newLogicalRouterPortCollector(client *nsxt.APIClient, logger log.Logger) prometheus.Collector {
+	rxTotalPacketDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "logical_router_port", "rx_total_packet"),
+		"Total packets of logical router port rx",
```

Comment:
nit: might be better to include rx after total or even use the english word, because it can be understood easily, i.e.
```
Total received (rx) packets of logical router port
```
same comments for the other metrics

Reply:
English word would be good, which one should we use?
Should we change all rx/tx with transmit/receive?
In metric's help and name?

Reply:
how about both for help:
received (rx)
transmitted (tx)

but name we can keep rx/tx because it's from NSX API?

Reply:
Updated help for better clarity